test/Prelude.purs

```diff
@@ -396,5 +396,11 @@ escapePathInErrMsg = case Process.platform of
 assertWarning :: forall m. MonadThrow Error m => Array String -> Boolean -> String -> m Unit
 assertWarning paths shouldHave stdErr  = do
```

Comment:
This function is super confusingly named for what it does and has confusing parameters too. I fixed the error message at least, but refactoring the function would be a bigger undertaking.